database.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TABLE brands (
+  id bigserial,
+  name text NOT NULL UNIQUE,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE products (
+  id bigserial,
+  brand_id bigint NOT NULL,
+  category_id bigint NOT NULL,
```

Comment:
I think you also should add fk for the category relation (same as for brands)

Reply:
Ok, fixed